Makefile

```diff
@@ -0,0 +1,25 @@
+generate:
+	protoc \
+		-I versionpb \
+		-I third_party/grpc-gateway/ \
+		-I third_party/googleapis \
+		--go_out=plugins=grpc,paths=source_relative:./versionpb \
+		--grpc-gateway_out=./versionpb \
+		--openapiv2_out=third_party/OpenAPI/ \
+		versionpb/version.proto
+
+	mv ./versionpb/github.com/Percona-Lab/percona-version-service/proto/* ./versionpb/
+	rm -r ./versionpb/github.com
+
+	statik -m -f -src third_party/OpenAPI/
+
+.PHONY: install
+install:
+	go get \
+		github.com/golang/protobuf/protoc-gen-go \
+		github.com/grpc-ecosystem/grpc-gateway/v2/protoc-gen-grpc-gateway \
+		github.com/grpc-ecosystem/grpc-gateway/v2/protoc-gen-openapiv2 \
+		github.com/rakyll/statik
```

Comment:
Tools can be (and should be, IMHO) versioned by moving them into own module. Example:
* https://github.com/percona-platform/dbaas-api/tree/main/tools
* https://github.com/percona-platform/dbaas-api/blob/dfd511a997633a225c78a294bd2ad49c692beaf9/Makefile#L12-L17

Reply:
https://github.com/Percona-Lab/percona-version-service/pull/1/commits/d3cd9c4a8b09c8fe337b16ce64236639dce8fe0e